src/components/Message/MessageEditedTimestamp.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import clsx from 'clsx';
+import { useMessageContext, useTranslationContext } from '../../context';
+import { DefaultStreamChatGenerics } from '../../types';
+import { MessageTimestampProps } from './MessageTimestamp';
```

Comment:
```suggestion
import type { DefaultStreamChatGenerics } from '../../types';
import type { MessageTimestampProps } from './MessageTimestamp';
```

Reply:
Maybe it's time to bump TS and enable `verbatimModuleSyntax` - I'll see if that's an easy pick :)